Yoast/YoastSEO.js
# Add paragraph structure

## Summary

This PR can be summarized in the following changelog entry:

* [not user-facing] Adds a tree-structure element for Paragraph.

## Relevant technical choices:

* Contrary to the original suggestion by Anton, I added opening and closing tags as parameters to the constructor, for flexibility.

## Test instructions

This PR can be tested by following these steps:

* Run the `yarn test` and see that nothing fails.

Fixes #2017
